core/src/main/scala/kafka/log/ProducerIdMapping.scala

```diff
@@ -261,7 +261,7 @@ class ProducerIdMapping(val config: LogConfig,
         case None =>
```

Comment:
Changed the `file.delete` a couple of lines above to `Files.delete` in a try/catch where we log if the delete fails.